docs/source/error_code_list.rst

```diff
@@ -665,6 +665,24 @@ consistently when using the call-based syntax. Example:
     # Error: First argument to namedtuple() should be "Point2D", not "Point"
     Point2D = NamedTuple("Point", [("x", int), ("y", int)])
 
+
+
+Implicitly converting to bool always evaluates as True [implicit-bool]
```

Comment:
I'm not sure whether "implicit-bool" should be the code, and whether "implicit conversion" captures what's happening here. Perhaps there's some accepted terminology for this in cpython?

Reply:
The Python docs seem to use the term "boolean operations"

Reply:
"context of Boolean operation" is also used